src/librbd/CMakeLists.txt

```diff
@@ -345,7 +345,7 @@ endif()
 if(ENABLE_SHARED)
   set_target_properties(librbd PROPERTIES
     OUTPUT_NAME rbd
-    VERSION 1.19.0
+    VERSION 1.19.3
```

Comment:
@idryomov ^ is this okay? We've never done a release bump like this before TMK.

Reply:
I don't have the context for "pre-kickoff".

For tentacle I'd expect it to be 1.20.0.  For a temporary workaround in main for something, this version could probably be left is as (i.e. 1.19.0).

Reply:
This "pre-kickoff" is just minimal changes necessary to get upgrades working from `squid` to `main`. (There are barriers in teuthology and possibly other places which prevent upgrades that appear to go backwards.)

Reply:
@idryomov also as a meta comment: would it not be better for this version to be pulled from the top-level `CMakeLists.txt`?

Reply:
Probably not.  It might make more sense to pull it from `src/include/rbd/librbd.h` -- it corresponds to `LIBRBD_VER_MAJOR`, `LIBRBD_VER_MINOR` and `LIBRBD_VER_EXTRA` there.

Note that we only change the minor there (the second component) and that nothing changes in stable branches (i.e. 1.19.0 would be the version both the initial squid release and all squid backport releases).